web_pwa_oca/static/src/js/worker/pwa.js

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2020 Tecnativa - Alexandre D. Díaz
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+"use strict";
+
+var PWA = OdooClass.extend({
```

Comment:
I see this file is not an odoo module. I guess it's a need for this module, but since this is awkward for the odoo developer, could you add some docs here explaining why is this different?